flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/sink/TestFlinkIcebergSink.java

```diff
@@ -326,4 +327,51 @@ public void testTwoSinksInDisjointedDAG() throws Exception {
     Assert.assertEquals("rightTable", rightTable.currentSnapshot().summary().get("direction"));
   }
 
+  @Test
+  public void testOverrideWriteConfigWithUnknownDistributionMode() {
+    AssertHelpers.assertThrows("Override write configuration.",
+            RuntimeException.class, "No enum constant org.apache.iceberg.DistributionMode.UNRECOGNIZED",
```

Comment:
Indentation in this class is off. Can you use 4 spaces for continuation indents?